ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -467,7 +469,7 @@ class MSFNotificationStateImpl: NSObject, ControlConfiguration, MSFNotificationS
          actionButtonAction: (() -> Void)? = nil,
          showDefaultDismissActionButton: Bool? = nil,
          messageButtonAction: (() -> Void)? = nil,
-         showFromBottom: Bool = true) {
+         showFromBottom: Bool? = nil) {
```

Comment:
why we cahnging this?

Reply:
was fixing an issue where SwiftUI's showFromBottom was hardcoded as "true" and went ahead and aligned the parameter with the other parameters in the inits

Reply:
I kinda want to push back on this: I don't think optional Bool values are generally a good design pattern, as it's ambiguous what the default behavior will be. Keeping it non-optional and just setting the default value to `false` seems easier to understand, unless there's a convention/pattern out there that I'm not familiar with.